providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -907,7 +907,7 @@ def _get_results(
         with contextlib.closing(self.get_conn(schema)) as conn, contextlib.closing(conn.cursor()) as cur:
             cur.arraysize = fetch_size or 1000
 
-            db = self.get_connection(self.hiveserver2_conn_id)  # type: ignore
+            db = self.get_connection(self.hiveserver2_conn_id)
```

Comment:
same here... as above
```suggestion
            db = self.get_connection(self.get_conn_id())
```